### PR TITLE
[SAIC-213] Replace current knife based hardware info collector

### PR DESCRIPTION
**Pull request by Settis**
Knife works only with Chef deployed cloud. For more general cases we must use Nova API for get hardware information.

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

File: condensation/scripts/nova_collector.py [hunk 1]
```diff
@@ -21,6 +21,9 @@
 tenant = ""
 auth_url = ""
 
+core_ratio = 1
+ram_ratio = 1
```
Comment (roman-verchikov) [inline condensation/scripts/nova_collector.py]: `ram_ratio` and `core_ratio` must be taken from config

File: condensation/scripts/nova_collector.py [hunk 2]
```diff
@@ -41,3 +44,21 @@
...
+
+for hypervisor in cli.hypervisors.list():
+    host = hypervisor.hypervisor_hostname
+    if host not in down_hosts:
+        result_dict.update(
+            {host: {
+                'core': hypervisor.vcpus,
+                'ram': hypervisor.memory_mb,
+                'core_ratio': 1,
+                'ram_ratio': 1}})
```
Comment (roman-verchikov) [inline condensation/scripts/nova_collector.py]: You should be using the values `core_ratio` and `ram_ratio` defined above

**Comment by roman-verchikov** [comment]
Put description to the pull request please

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

**Comment by roman-verchikov** [comment]
Please rebase and update description

**Comment by Settis** [comment]
Update it.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSED.

**Comment by roman-verchikov** [comment]
+1

File: condensation/scripts/__init__.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2015 Mirantis Inc.
```
Comment (miarmak) [inline condensation/scripts/__init__.py]: There is no need to put copyrights to the `__init__.py` files, so it can be removed as well.

**Comment by miarmak** [comment]
And also, there is no need to have information about conflicts appeared during the rebase process in the commit description. It can be removed too.
Everything else looks fine, thank you.

**Comment by roman-verchikov** [comment]
I would assess @miarmak's comments as minor, so this can be merged.

**Pull request merged**